test/functional/test_framework/test_framework.py

```diff
@@ -2218,10 +2219,12 @@ def mine_cycle_quorum(self, is_first=True):
 
         skip_count = cycle_length - (cur_block % cycle_length)
         # move forward to next 3 DKG rounds for the first quorum
-        extra_blocks = 24 * 3 if is_first else 0
+        extra_blocks = 0 if self.cycle_quorum_is_ready else 24 * 3
```

Comment:
let's extract this 24 to be non-magic @coderabbitai, is there a spot that defines the cycle length we can reuse?